packages/mattermost-redux/src/selectors/entities/users.ts

```diff
@@ -681,3 +684,15 @@ export function searchProfilesInGroup(state: GlobalState, groupId: $ID<Group>, t
 
     return profiles;
 }
+
+export const isCurrentUserNewAccount = createSelector(
+    'isCurrentUserNewAccount',
+    (state: GlobalState) => getCurrentUser(state),
+    (currentUser): boolean => {
+        const today = moment(Date.now());
+        const created = moment(new Date(currentUser.create_at));
+        const d = today.diff(created, 'days');
+        const existedForSomeTime = d > NUMBER_OF_DAYS_CONSIDERED_NEW_USER;
```

Comment:
Try and use audit tables to see last login. See here https://community.mattermost.com/core/pl/yoad19cwqj89in5rs5h3iswf3a